js/src/index.js

```diff
@@ -227,7 +243,6 @@ export default class FlowApp {
 
     const payload = Buffer.from([slotIdx]);
     return this.transport.send(CLA, INS.GET_SLOT, 0, 0, payload).then((response) => {
-      console.log(response);
```

Comment:
Why are we removing so many log statements in this file?

Reply:
I think these log commands are remains of some debugging done way in the past. The few log commands are too few and too random to make sense as desired feature. Also, this code is supposed to work with a variety of clients, I do not think, console.log is appropriate.